serial_driver/package.xml

```diff
@@ -15,7 +15,10 @@
   <buildtool_depend>asio_cmake_module</buildtool_depend>
 
   <depend>asio</depend>
+  <depend>io_context</depend>
   <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>rclcpp_lifecycle</depend>
   <depend>std_msgs</depend>
```

Comment:
should there also be `example_interfaces` listed here?

Reply:
Since this package depends on `io_context` and `io_context` depends on `example_interfaces`, it isn't strictly required. `ament` populates CMake include and library lists recursively.